ribasim_qgis/widgets/dataset_widget.py

```diff
@@ -260,8 +260,8 @@ def load_geopackage(self) -> None:
 
     def _get_database_path_from_model_file(self) -> str:
         with open(self.path, "rb") as f:
-            model_filename = tomllib.load(f)["database"]
-            return str(Path(self.path).parent.joinpath(model_filename))
+            input_dir = tomllib.load(f)["input_dir"]
+        return str((Path(self.path).parent / input_dir / "database.gpkg").resolve())
```

Comment:
Is `input_dir` always relative? Perhaps perform some more checks before you just append the paths. @visr what are the agreements on `input_dir`?

Reply:
It can be absolute as well, but `/` handles that case as desired, removing the preceding part.

Reply:
I tested it manually to be sure, I learned something today

Reply:
Oh nice